src/components/VmDialog/index.js

```diff
@@ -316,6 +324,34 @@ class VmDialog extends React.Component {
     }
   }
 
+  checkTimeZone ({ osType }) {
+    let timeZone = null
+    const template = this.getTemplate()
+    if (template) {
+      if (template.getIn(['timeZone', 'name'])) {
+        timeZone = timeZone || template.get('timeZone').toJS()
+        const isWindowsTimeZone = timezones.find(timezone => timezone.id === timeZone.name)
+        const isWindowsVm = isWindows(osType)
+
+        if (isWindowsVm && !isWindowsTimeZone) {
+          timeZone = {
+            name: 'GMT Standard Time',
```

Comment:
We should not hard-coded the timezone defaults since they are declared as config values  "vdcDefaultGeneralTimeZone", "DefaultWindowsTimeZone" on vdc_options table.
I suggest that in case we want to set the default then set the timeZone to null and let the engine backend choose the value for you as set in https://github.com/oVirt/ovirt-engine/blob/2cc0ab0ff6b00b5b51975497eaf1747b29d2a69c/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VmHandler.java#L800.